skimage/filters/_frangi.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+
+__all__ = ['frangi', 'hessian']
+
+
+def filtered_image_calculation(image, scale, scale_step, beta1, beta2,
```

Comment:
Indicate that this is a "private" function by starting with an underscore. Also, the name is a little bit nondescript... I need to think of a better name for this...